product_packaging_level/models/product_packaging.py

```diff
@@ -116,16 +120,17 @@ def _make_qty_per_level(self, qty_per_level_mapping, format_pattern=None):
             res.append((code, new_qty))
         return res
 
-    @api.onchange("packaging_level_id")
-    def _onchange_name(self):
-        if self.packaging_level_id:
-            self.name = self.packaging_level_id.name
+    @api.onchange("package_type_id")
+    def _onchange_package_type(self):
+        package_type_level_id = self.package_type_id.packaging_level_id
+        if package_type_level_id:
+            self.packaging_level_id = package_type_level_id
 
-    def name_get(self):
-        result = []
-        for record in self:
-            if record.product_id and record.packaging_level_id:
-                result.append((record.id, record.packaging_level_id.display_name))
-            else:
-                result.append((record.id, record.name))
-        return result
+    @api.onchange("packaging_level_id", "package_type_id", "name", "name_policy")
+    def _onchange_name(self):
```

Comment:
@anothingguy @jbaudoux The change from `name_get` to an `onchange` is very annoying. It breaks my use of this addon. Moreover, the name on the package level is `translate=True` but not on the product packaging, that was one of the motivations to override the `name_get` .

Reply:
cc @simahawk

Reply:
So we should 
* recover name_get, return value according to name policy and add a comment that this is to respect translations on level name
* replace _onchange_name by making the name computed stored=True

then maybe also change _onchange_package_type by making packaging_level_id computed store=True based on package_type_id

correct ?

Reply:
@lmignon Fixed here https://github.com/OCA/product-attribute/pull/1527 Can you review?